src/App.jsx

```diff
@@ -3,31 +3,34 @@ import reactLogo from './assets/react.svg'
 import viteLogo from '/vite.svg'
 import './App.css'
 
+const todoList = [
+  {
+    id: 1,
+    title: 'Complete week 1 assignment'
+  },
+  {
+    id: 2,
+    title: 'Complete week 2 assignment'
+  },
+  {
+    id: 3,
+    title: 'Complete week 3 assignment'
+  }
+];
+
 function App() {
   const [count, setCount] = useState(0)
 
   return (
     <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+      <h1>Todo-List</h1>
+      <ul>
+        {todoList.map((todo) => {
+          return (
+            <li>{`${todo.id}) ${todo.title}`}</li>
```

Comment:
2) When you submit the hometask please make sure there are no errors in the console and in the terminal. 
Please fix it and re-request my review.
In case you have issues fixing, [here is a link to help you](https://codechronicle.medium.com/how-to-handle-warning-each-child-in-a-list-should-have-a-unique-key-prop-error-e9e6e3892a39#:~:text=The%20“Warning%3A%20Each%20child%20in,added%2C%20modified%2C%20or%20removed.).
<img width="1176" alt="Screenshot 2024-06-17 at 10 01 09" src="https://github.com/omk963/react-todo/assets/114008959/4dffb680-b59f-4718-8bf8-5a21efb6af39">